### PR TITLE
TMA-969 brick does not ends when run with nonsensical delete params (…

### PR DESCRIPTION
**Pull request by Hahihula**
…or mode)

### CODE REVIEW

File: spec/unit/actions/associate_clients_spec.rb [hunk 1]
```diff
@@ -92,4 +92,32 @@
       expect { GoodData::LCM2.run_action(GoodData::LCM2::AssociateClients, params) }.to raise_error(/The parameter/)
     end
   end
+
+  context 'in any delete mode' do
+    let(:params) do
```
Comment (yenkins) [inline spec/unit/actions/associate_clients_spec.rb]: Identical code found in associate_clients_spec.rb:36, associate_clients_spec.rb:97 (mass = 76)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/284/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1131/ : FAILURE in 1m 13s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1401/ : SUCCESS in 2m 31s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/288/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1135/ : FAILURE in 1m 18s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1405/ : SUCCESS in 1m 05s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/289/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1136/ : SUCCESS in 2m 31s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1406/ : SUCCESS in 2m 25s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/294/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1141/ : SUCCESS in 2m 43s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1411/ : SUCCESS in 2m 32s

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/90/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1142/ : SUCCESS in 2m 38s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1412/ : SUCCESS in 2m 31s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1451/ : SUCCESS in 14m 57s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/22/ : FAILURE in 40m 31s

**Comment by kubamahnert** [comment]
Merging as the result of the test was a misreported success:

12:57:52 appstore_1    | 88 examples, 0 failures, 1 pending, 1 error occurred outside of examples

**Pull request merged**